### DISCUSSION TITLE
Phantom??

### DISCUSSION THREAD

**Discussion by Amatheian** [winstxnhdw/lc-hax#429] [Q&A]
so Phantom mode bind is "=" right? and I m pressing LShift+0, also i tried RShift+0 but its not working.

**Comment by winstxnhdw** [comment]
There’s no modifier for =. What keyboard are you using?

**Reply by Amatheian** [reply]
rampage slayer

**Reply by winstxnhdw** [reply]
That’s not what I meant 😭😭😭 As in what layout..

**Reply by Amatheian** [reply]
qwerty

**Reply by winstxnhdw** [reply]
US? UK?

**Reply by Amatheian** [reply]
TR

**Reply by Amatheian** [reply]
so what now? do i need to change keybinds?

**Reply by winstxnhdw** [reply]
Can you even type = here?

**Reply by winstxnhdw** [reply]
Or are you copy-pasting?

**Reply by Amatheian** [reply]
= yeah but i need to press 2 keys to get this, its a shortcut like two key combination

**Reply by Amatheian** [reply]
no m not doing that

**Reply by winstxnhdw** [reply]
and that combination doesn’t work?

**Reply by Amatheian** [reply]
yeah its not working because you know its not an actual key for me i need to press two keys

**Reply by winstxnhdw** [reply]
I guess you can rebind it to another key. There’s a discussion where I showed a guy how.

**Reply by Amatheian** [reply]
i saw that earlier but i can't figure it out which one i need to change its so complicated

**Reply by winstxnhdw** [reply]
Pretty sure I even linked him the exact line he has to change.

**Reply by Amatheian** [reply]
no its not like that 
(() => Keyboard.current[Key.Digit0].wasPressedThisFrame, () => InputListener.OnEqualsPress?.Invoke()), 
NO its not working. You need to do this instead: 
(() => Keyboard.current[Key.Digit0].wasPressedThisFrame, () => InputListener.OnDigit0Press?.Invoke()), 

also in Modules>PhantomMod.cs you need to change the lines to this;

void OnEnable() {
        ..
        InputListener.OnDigit0Press += this.TogglePhantom;
        ..

    void OnDisable() {
        ..
        InputListener.OnDigit0Press -= this.TogglePhantom;
        ..
        
        and i figure it out myself and it worked

**Reply by winstxnhdw** [reply]
Congratulations, you just broke every future bind that may use `=`.

This is fine and will work. `OnEqualsPress` is just the variable name defined by me. Nothing special about it.

```cs
(() => Keyboard.current[Key.Digit0].wasPressedThisFrame, () => InputListener.OnEqualsPress?.Invoke())
```
